src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatUtil.java

```diff
@@ -60,5 +65,26 @@ static LifecycleListener getDefaultWebXmlListener() {
         return Arrays.copyOfRange(classContext, 2, classContext.length);
     }
 
+    static boolean isJar(Path path) {
+        if (!Files.isRegularFile(path)) {
+            return false;
+        }
+
+        try (JarFile jar = new JarFile(path.toFile())) {
+            // Quick sanity check: should contain at least one file.
+            int numEntries = 0;
+            for (Enumeration<JarEntry> i = jar.entries(); i.hasMoreElements();) {
+                final JarEntry e = i.nextElement();
+                if (!e.isDirectory()) {
+                    numEntries++;
```

Comment:
Yes, I wanted to check if all entries are OK, but I guess it's not the responsibility of this method. Let me update it.